app/models/customer.rb

```diff
@@ -29,11 +29,12 @@ class Customer < ApplicationRecord
 
   sequenced scope: ->(customer) { customer.organization.customers }
 
-  validates :external_id, presence: true, uniqueness: { scope: :organization_id }
   validates :country, country_code: true, unless: -> { country.nil? }
-  validates :vat_rate, numericality: { less_than_or_equal_to: 100, greater_than_or_equal_to: 0 }, allow_nil: true
-  validates :payment_provider, inclusion: { in: PAYMENT_PROVIDERS }, allow_nil: true
   validates :currency, inclusion: { in: currency_list }, allow_nil: true
+  validates :external_id, presence: true, uniqueness: { scope: :organization_id }
```

Comment:
Rails/UniqueValidationWithoutIndex: Uniqueness validation should have a unique index on the database column.

Reply:
Interesting - Will fix that in a separate PR 👍